src/components/NcListItem/NcListItem.vue

```diff
@@ -320,28 +331,30 @@
 		:exact="to ? exact : null">
 		<li class="list-item__wrapper"
 			:class="{ 'list-item__wrapper--active' : isActive || active }">
-			<a :id="anchorId"
-				ref="list-item"
-				:href="routerLinkHref || href"
-				:target="target || (href === '#' ? undefined : '_blank')"
-				:rel="href === '#' ? undefined : 'noopener noreferrer'"
+			<div ref="list-item"
 				class="list-item"
-				:aria-label="linkAriaLabel"
+				tabindex="0"
 				@mouseover="handleMouseover"
 				@mouseleave="handleMouseleave"
 				@focus="handleFocus"
 				@blur="handleBlur"
 				@keydown.tab.exact="handleTab"
 				@click="onClick($event, navigate, routerLinkHref)"
 				@keydown.esc="hideActions">
-
-				<div class="list-item-content__wrapper"
-					:class="{ 'list-item-content__wrapper--compact': compact }">
+				<div :class="['list-item-content__wrapper', {
+					'list-item-content__wrapper--compact': compact
+				}]">
 					<!-- @slot This slot is used for the NcAvatar or icon -->
 					<slot name="icon" />
 
 					<!-- Main content -->
-					<div class="list-item-content">
+					<a :id="anchorId || undefined"
```

Comment:
Child divs need to be switched to spans as `<a>` is inline

Reply:
Are you sure? I read the specs that every content is allowed within `a` except for interactive content and such with `tabindex` specified.

Reply:
Link to specs?

Reply:
If HTML validator passes then fine with me :)

Reply:
Either: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a?retiredLocale=de#technical_summary
Or https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element